settings.gradle

```diff
@@ -3,6 +3,7 @@ include ':api'
 include ':basicdataset'
 include ':examples'
 include ':model-zoo'
+include ':easy'
```

Comment:
It's a bit strange call a module easy. may call it starter

Reply:
I was thinking of it similarly to EasyTrain (DJL-Easy in full).

I don't know if I like starter, because this isn't necessarily the start to using DJL. It is a higher level of abstraction over all of the deep learning provided by DJL. The goal is not to introduce them to DJL concepts, but to hide as many concepts as possible. Furthermore, I am afraid that referring to it as a starter might discourage others from treating it as a serious product. Ultimately, I think we can expect many users of DJL-easy to never have the time/inclination to learn enough deep learning to produce better results than DJL-Easy and to just keep using it indefinitely.

Reply:
I agree starter may not be a good name, but, '`easy` is too general, it didn't provide enough context. We can discuss a little bit more. Here is some name I can think of:
- simple-train
- djl-simple
- easy-train
- 4dummies